src/pykeen/nn/init.py

```diff
@@ -109,3 +111,27 @@ def init_quaternions(
     imag = imag * (modulus * phase.sin()).unsqueeze(dim=-1)
     x = torch.cat([real, imag], dim=-1)
     return x.view(num_elements, 4 * dim)
+
+
+def create_init_from_pretrained(
+    pretrained: torch.FloatTensor,
+) -> Initializer:
+    """
+    Create an initializer via a constant vector.
+
+    :param pretrained:
+        the tensor of pretrained embeddings.
+
+    :return:
+        an initializer, which fills a tensor with the given weights.
+    """
+
+    def init_from_pretrained(
```

Comment:
having internal functions is a problem for pickling. we should first update the unit tests to reflect these cases, then the fix is to refactor this into a class (like our pykeen.utils.compose utility)

Reply:
I'll take care of it

Reply:
Renaming this method to `__call__`, and making the factory method around a class should fix it :+1:

Reply:
I added saving/loading the model to the tests and this seems to not actually be a problem